### DISCUSSION TITLE
Download Build Output

### DISCUSSION THREAD

**Discussion by definoob** [vercel/vercel#6740] [Help]
![1](https://user-images.githubusercontent.com/54032883/134026660-c719b794-3996-4079-b9e4-468387aaa7db.png)

So I found the output of my project build but is there a way to download these things to my machine?

Thanks.

**Comment by delbaoliveira** [comment]
Hey @definoob 👋🏼 

Do you mind sharing your use case or what you're trying to achieve?

**Comment by definoob** [comment]
I have got some problem
So the code working on my localhost, deployed it and staging deployment doesn't work.
That's why I want to download the build output and test on my local.

**Reply by delbaoliveira** [reply]
Thank you!

Are you using a framework? And are there any errors in the build log or the function logs?

**Reply by definoob** [reply]
This is Next.JS. and I don't have got any errors. 
So lemme explain more. 
When I said <doesn't work>, this is a dApp and can't connect to metamask on android phone.
But the thing is
- Main Branch: working
- Main Branch Deployment: working
- Staging Branch: working
- Staging Branch Deployment: NOT WORKING

So I just want to download the build output and see if I can get a clue there.
